src/RA_RichPresence.cpp

```diff
@@ -241,7 +239,9 @@ bool RA_RichPresenceInterpreter::Load(ra::services::TextReader& pReader)
                 newLookup.AddLookupData(nVal, sLabel);
             } while (true);
 
+#ifndef RA_UTEST
             RA_LOG("RP: Adding Lookup %s (%zu items)\n", sLookupName.c_str(), newLookup.NumItems());
+#endif /* !RA_UTEST */
```

Comment:
`RA_Log.h` already undefines these for `RA_UTEST`. You shouldn't have to do this here, or we'd have to do it for all the files that get pulled into the Tests project.